code/game/area/areas.dm

```diff
@@ -107,6 +103,51 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 	uid = ++global_uid
 	related = list(src)
 	map_name = name // Save the initial (the name set in the map) name of the area.
+	if(!ambientsounds)
```

Comment:
Put this in a wrapper. 

Also i think expanding the tags into list values would be more useful here. 
As in `list(HOLY,'addtionalsound.ogg')` so you can expand on the defaults more easily.